samples/scaleout/senderside-and-distributor/Core_6/Shared/PlaceOrder.cs

```diff
@@ -0,0 +1,7 @@
+﻿using System;
+
+public class PlaceOrder :
```

Comment:
IMessage is only used for the custom convention. This is not the point of this sample.